### DISCUSSION TITLE
Help for twitch? TWITCH_REFRESH_TOKEN (resolved)

### DISCUSSION THREAD

**Discussion by avesanidennis** [jef/streetmerchant#1667] [Help] [answered]
I managed to put in all the codes except this one: TWITCH_REFRESH_TOKEN

RESOLVED: https://github.com/jef/streetmerchant/wiki/Notification:-Twitch

**Comment by agwosdz** [comment]
Awesome, 

I have also figured it out and written an automated js script to get all the needed tokens in node.

1. Sign up for a bot at dev.twitch.tv
2. Populate TWITCH_CLIENT_ID, TWITCH_CLIENT_SECRET, and TWITCH_CHANNEL in dotenv
3. run the script

Please let me know if there is any interest in the script and I can post/send.

Zet

**Comment by avesanidennis** [comment, marked as answer]
It would be nice, thank you.

**Reply by agwosdz** [reply]
I put it up on 
[https://github.com/jef/streetmerchant/wiki/Notification:-Twitch-(automated)](https://github.com/jef/streetmerchant/wiki/Notification:-Twitch-(automated))
